examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -0,0 +1,910 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    from merlin.models.tf import Loader
</pre><p>Guess some imports repeats again? you can put all the imports in the first cell may be?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>good ploint, removed duplicates and moved it all to first cell</p>